pandas-stubs/core/series.pyi

```diff
@@ -1528,6 +1527,7 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
         min_periods: int | None = ...,
         center: _bool = ...,
         *,
+        win_type: None = ...,
```

Comment:
`win_type` is `_str | None`

Reply:
see https://github.com/pandas-dev/pandas-stubs/pull/313#discussion_r976472297